src/index.test.ts

```diff
@@ -455,3 +455,54 @@ test('enforces a specified object interface', async () => {
   expect(log.propertyNotAllowed).toBe('Linting error');
   expect(log.input).toStrictEqual({ foo: 0xf00 });
 });
+
+test('should not log timestamp if timestamp logger option is false', async () => {
+  const stream = sink();
+  const logger = createLogger(
+    {
+      timestamp: false,
+    },
+    stream,
+  );
+
+  logger.info<ExampleMessageContext>(
+    {
+      activity: 'Testing Logger',
+      // @ts-expect-error
+      propertyNotAllowed: 'Linting error',
```

Comment:
This does not seem relevant to the test case:

```suggestion
```

Reply:
Good catch! @72636c